content/ru/guide/v11/upgrade-guide.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Обновление с Preact 8.x
+description: Узнайте, как обновить приложение Preact 8.x до Preact X
```

Comment:
This doc needs to be rewritten for v10 -> v11, see https://github.com/preactjs/preact-www/blob/master/content/en/guide/v11/upgrade-guide.md